src/app/errorPageStyling.tsx

```diff
@@ -0,0 +1,31 @@
+import Paper from "@mui/material/Paper";
```

Comment:
Just want to know if you need a use client on this page?

Reply:
Nope as it is just a styling page, can't see why I would?

Reply:
yeah i was confused yesterday for a second, but you need use client for styled components